src/routes/docs/tooling/command-line/non-interactive/+page.markdoc

```diff
@@ -3,21 +3,36 @@ layout: article
 title: Non-interactive
 description: Deploy changes to Appwrite projects to migrate databases and collections schema, functions, teams, buckets, and more.
 ---
-The Appwrite CLI can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server. You can enable the non-interactive mode for the Appwrite CLI by setting the `project ID`, `endpoint`, and `API Key`:
+
+The [Appwrite CLI](/docs/tooling/command-line/installation) can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a continuous integration server.
+
+# Authentication {% #authentication %}
+
+In non-interactive mode, you can authenticate your users using two methods: email and password. While using an [API key](/docs/advanced/platform/api-keys) does not require an email and password to authenticate users, you need to configure the API key [scopes](/docs/advanced/platform/api-keys#scopes) of the application and does not work for project API calls that update project settings.
```

Comment:
> In non-interactive mode, you can authenticate your users using two methods: email and password.

Let's remove the "your users" part because you're not really authenticating as a user. Also, what happened to the API key method?

> While using an [API key](/docs/advanced/platform/api-keys) does not require an email and password to authenticate users

Don't mention users because the authentication is not as an end user.

> and does not work for project API calls that update project settings.

you need to configure [...] and does not work [...] doesn't sound right.